.github/workflows/e2e-tests.yml

```diff
@@ -44,12 +44,12 @@ jobs:
         with:
           repository: "${{ github.repository_owner }}/kubewarden-end-to-end-tests"
           path: "e2e-tests"
-          ref: "v0.2.3"
+          ref: 'main'
```

Comment:
I would prefer to use always latest version of tests, does it make sense to you?
Not sure if there is a reason for tagging them.

Reply:
I see the benefit and easy of use of using `main`. 

I think the main reason to use versioning is to allow incomplete things in the main devel branch and the standard way of releasing software. xD

If based on your experience this is an overkill, we can start to use `main`. We can always change again when the need shows up. ;)